src/python/national_voter_file/us_states/nc/transformer.py

```diff
@@ -83,7 +83,7 @@ def extract_do_not_call_status(self, input_dict):
 
     def extract_gender(self, input_dict):
         gender = input_dict['gender_code']
-        if len(gender) == 0:
+        if len(gender) == 0 or gender == ' ':
```

Comment:
we're moving to None for non-entry for most fields instead of these magic strings, should he the same here